Task_4/Operation.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_4
+{
+    public abstract class Operation
+    {
+        protected double FirstOperand { get; set; }
+
+        protected double SecondOperand { get; set; }
+
+        protected string Action { get; set; }
+
+        public Operation(double firstOperand, double secondOperand)
+        {
+            FirstOperand = firstOperand;
+            SecondOperand = secondOperand;
+        }
+
+        public void ToString()
```

Comment:
Now you are aware about ToString method from object class that could be overriden.